### PR TITLE
Closes #423 - Mentions c0d3.com being free. Rewords footer.

### LINKED ISSUE #423: c0d3.com will be free forever and we should say it somewhere on the landing page!
I was referring c0d3.com to a new software developer and he’s was wondering if it’s free then I have noticed we say nowhere it’s free on the landing page.

### PR DESCRIPTION
**Pull request by pmulard**
The original mentions starting 3 times in this little footer section, so I figured this would be a good spot to rework the phrases and add in c0d3.com being free. Below are before and after screenshots.

edit: I see now the CI/build failed because I forgot to run yarn auto fix on the test suits before committing the changes. Not sure exactly how to handle this... this is my first pull request :)

image
image

### CODE REVIEW

**Comment by ggwadera** [comment]
Hey there, nice work fixing this. Thanks 🙂

For the failed tests, only the snapshot tests failed. It compares the rendered DOM to a pre-rendered file, since you modified the text it caused them to fail.

To fix it just run `yarn test -u`, this will generate new snapshots for the tests.

File: components/LandingPage.tsx [hunk 1]
```diff
@@ -188,12 +188,12 @@ const LandingPage: React.FC = () => {
...
       </div>
       <div className="container mt-5 py-5 text-center">
-        <h1 className="display-4 font-weight-bold">Get Started Now</h1>
+        <h1 className="display-4 font-weight-bold">Start Your Journey</h1>
         <p
           className="mt-3 font-weight-light"
           style={{ lineHeight: 1.5, fontSize: '1.5rem' }}
         >
-          Start your journey to being a full stack software engineer
+          Become a full stack software engineer - 100% Free
```
Comment (anthonykhoa) [inline components/LandingPage.tsx:196]: I think adding an exclamation mark at the end will make this message sound better. 
`Become a full stack software engineer - 100% Free!` 
I think adding the exclamation mark will give the message a more `enthusiastic` tone, while also creating more emphasis that c0d3 is 100% free.

Comment (pmulard) [reply components/LandingPage.tsx:196]: I agree, I like the enthusiasm!

**Comment by ggwadera** [comment]
You gotta commit the updated snapshots too! Otherwise the build system will use the current ones.

**Comment by pmulard** [comment]
I see now. Thanks for bearing with me!

**Review by ggwadera — APPROVED** [review]

**Comment by ggwadera** [comment]
> I see now. Thanks for bearing with me!

No worries, thank you for your contribution!

Let me see if the contributions bot still works haha.

@all-contributors please add @pmulard for code

**Comment by pmulard** [comment]
Sweet! First contribution is in the books! Here's to many more

**Pull request merged**